src/components/card/card.material.scss

```diff
@@ -4,26 +4,21 @@
 :host {
     display: flex;
     flex-direction: column;
-    box-shadow: var(--igc-elevation-4);
+    border: rem(1px) solid color(gray, 400);
     overflow: hidden;
     border-radius: rem(3px);
     background: color(surface, 500);
-    transition: box-shadow .1s ease-out;
-    backface-visibility: hidden;
     font-family: var(--igc-font-family);
 }
 
-:host(:hover),
-:host(:focus-within) {
-    box-shadow: var(--igc-elevation-6);
-}
-
-:host([outlined]) {
-    box-shadow: none;
-    border: rem(1px) solid color(gray, 400);
+:host([elevated]) {
+    border: none;
+    box-shadow: var(--igc-elevation-4);
+    backface-visibility: hidden;
+    transition: box-shadow .1s ease-out;
 
     &:hover,
     &:focus-within {
-        box-shadow: none;
+        box-shadow: var(--igc-elevation-6);
```

Comment:
The hover styling when elevated is set and the theme is material is not working. I was able to make it work like this:
```scss
:host([elevated]) {
    border: none;
    box-shadow: var(--igc-elevation-4);
    backface-visibility: hidden;
    transition: box-shadow .1s ease-out;
}

:host([elevated]:hover),
:host([elevated]:focus-within) {
    box-shadow: var(--igc-elevation-6);
}
```